gpu-operator/google-gke.rst

```diff
@@ -67,11 +67,115 @@ Prerequisites
   in the Google Cloud documentation.
 
 
-*********
-Procedure
-*********
+*********************************
+Using the Google Driver Installer
+*********************************
+
+Perform the following steps to create a GKE cluster with the ``gcloud`` CLI and use Google driver installer to manage the GPU driver.
+You can create a node pool that uses a Container-Optimized OS node image or a Ubuntu node image.
+
+#. Create the cluster.
+   Refer to `Running GPUs in GKE Standard clusters <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#create>`__
+   in the GKE documentation.
+
+   When you create the cluster, specify the following additional ``gcloud`` command-line options:
+
+   - ``--node-labels="gke-no-default-nvidia-gpu-device-plugin=true"``
```

Comment:
Question -- is this required for Ubuntu node pools? I only ask since our existing documentation, which only covers Ubuntu, does not mention this label.

Reply:
cc @Dragoncell 

I would expect we need it for Ubuntu as well (otherwise we will run the GKE GPU device plugin on those nodes).